src/web/components/Layout/Sidebar.jsx

```diff
@@ -9,8 +9,59 @@ import ReactSidebar from 'react-sidebar'
 import SidebarHeader from './SidebarHeader'
 import RulesChecker from '+/views/RulesChecker'
 
+import GhostChecker from '~/views/GhostContent/Checker'
+
 const style = require('./Sidebar.scss')
 
+const BASIC_MENU_ITEMS = [
+  {
+    name: 'Dashboard',
+    path: 'dashboard',
+    activePaths: ['', '/', '/dashboard'],
+    rule: { res: 'dashboard', op: 'read' },
+    icon: 'dashboard'
+  },
+  {
+    name: 'Modules',
+    path: 'manage',
+    activePaths: ['/manage'],
+    rule: { res: 'modules/list', op: 'read' },
+    icon: 'dashboard'
+  },
+  window.GHOST_ENABLED && {
+    name: 'Ghost Content',
+    path: 'ghost-content',
+    activePaths: ['/ghost-content'],
+    rule: { res: 'ghost_content', op: 'read' },
+    icon: 'content_copy',
+    renderSuffix: () => <GhostChecker />
+  },
+  {
+    name: 'Content',
+    path: 'content',
+    activePaths: ['/content'],
+    rule: { res: 'content', op: 'read' },
+    icon: 'description'
+  },
+  {
+    name: 'Flows',
+    path: 'flows',
+    activePaths: ['/flows'],
+    rule: { res: 'flows', op: 'read' },
+    icon: 'device_hub'
+  },
+  {
+    name: 'Middleware',
+    path: 'middleware',
+    activePaths: ['/middleware'],
+    rule: { res: 'middleware', op: 'read' },
+    icon: 'settings'
+  }
+].filter(Boolean)
+
+// const ummPaths = ['/umm']
+// const ummRules = { res: 'umm', op: 'read' }
```

Comment:
@emirotin , did you leave these commented lines intentionally?

Reply:
Yes, they were in the original code but were unused. We either planned to have this menu item but deleted it and left couple of variables. But then I forgot to ask about them :)
@slvnperron what's the case, should I delete these lines?

Reply:
Yes, we'll remove UMM ui entirely. Thanks